models/sklearn/classification/evaluation.py

```diff
@@ -0,0 +1,279 @@
+import os
+import json
+import numpy as np
+import pandas as pd
+import seaborn as sns
+import matplotlib.pyplot as plt
+from termcolor import colored
+from pprint import pprint
+import yaml
+from sklearn.model_selection import KFold
+from sklearn.metrics import accuracy_score
+from sklearn.metrics import confusion_matrix, classification_report
+import joblib
+import requests
+from utils import load_yaml, FindCreateDirectory, TrainingProcesses
+from transformation.transform import Transform
+from transformation.utils_preprocessing import flatten_dict_full
+from classification.report_files_export import export_report
+from logging_tool import LoggerSetup
+
+
+def fold_evaluation(config, n_fold, X, y, class_name, tracks, process, exports_path, log_level):
+    # logger setup
```

Comment:
this kind of comment isn't necessary in this code. We know that it's logger setup because that's the name of the class that you're calling!